tracing-honeycomb/Cargo.toml

```
@@ -26,6 +26,8 @@ chrono = "0.4"
 parking_lot = { version = "0.11", optional = true }
 uuid = { version = "0.8", features = ["v4"] }
 sha-1 = "0.9"
+serde = "1.0.114"
+serde_json = "1.0.56"
```

We should probably be less specific here
```suggestion
serde_json = "1"
```